src/Persister/ElasticSearchPersister.php

```diff
@@ -73,16 +76,17 @@ public function __construct($options = [])
     }
 
     /**
-     * Persists all of the audit log event objects that are provided.
+     * Persists all the audit log event objects that are provided.
      *
      * @param array $auditLogs An array of EventInterface objects
      * @return void
      */
     public function logEvents(array $auditLogs)
     {
-        $client = $this->getConnection();
         $documents = $this->transformToDocuments($auditLogs);
 
+        $connection = $this->getConnection();
+        $client = $connection->getDriver();
         $client->addDocuments($documents);
```

Comment:
This might need someone who is actually using elastic to look at it, but it passes the pipeline.